panel/src/components/Navigation/Topbar.vue

```diff
@@ -144,27 +144,18 @@
 </template>
 
 <script>
-import views from "@/config/views.js";
-
 export default {
+  props: {
+    areas: Object,
+    breadcrumb: Array,
+    license: [Boolean, String],
```

Comment:
This should always be `Boolean` because of Bastian's change. We can also simplify the `unregistered()` computed value or remove it and use `!this.license` directly.